docs/getting-started-scalardl-ledger.md

```diff
@@ -0,0 +1,538 @@
+# Getting Started with Helm Charts (Scalar DL Ledger)
+
+This document explains how to get started with Scalar DL Ledger by using the Helm Chart in your test environment. Here, we assume that you already have a Mac or Linux environment for testing.  
+
+## Requirement
+
+* You need the privileges to pull the Scalar DL containers (`scalar-ledger` and `scalardl-schema-loader`) from [GitHub Packages](https://github.com/orgs/scalar-labs/packages).  
+* You must create a Github Personal Access Token (PAT) with `read:packages` scope according to the [GitHub document](https://docs.github.com/en/github/authenticating-to-github/keeping-your-account-and-data-secure/creating-a-personal-access-token), to pull the above containers.
+
+## Tools
+
+In this guide, we will use the following tools for testing.  
+
+1. Docker
+1. minikube
+1. kubectl
+1. Helm
+1. cfssl / cfssljson
+
+## Environment
+
+In this guide, we will create the following environment in your local by using Docker and minikube.  
+
+```
+On the Docker Network (named minikube)
+
+                   +-----------------------+
+  +-----------+    | +------------------+  |    +-----------+
+  |           |    | | Scalar DL Ledger |  |    |           |
+  |  Client   |    | +------------------+  |    | Cassandra |
+  | Container |    |                       |    | Container |
+  |           |    | Kubernetes (minikube) |    |           |
+  +-----+-----+    +-----------+-----------+    +-----+-----+
+        |                      |                      |
+*-------+----------------------+----------------------+-------*
+             ------->                     ------->
+               gRPC                    Cassandra Driver
+```
+
+
+## Step 1. Install tools
```

Comment:
Can we keep it as a separate document for use in Scalar DB and Scalar DL?
We can avoid redundancy.